kore/src/Kore/Step/Simplification/ExpandAlias.hs

```diff
@@ -66,18 +67,18 @@ expandSingleAlias
     -> Maybe (TermLike variable)
 expandSingleAlias =
     \case
-        ApplyAlias_ alias children -> pure $ substituteWorker alias children
+        ApplyAlias_ alias children -> pure $ substituteInAlias alias children
         _ -> Nothing
 
 -- TODO(Vladimir): Check aliases such that the intersection of alias variables
 -- with the *bounded* variables in the rhs is empty (because we can't currently
 -- handle things like \mu.
-substituteWorker
+substituteInAlias
```

Comment:
Would `expandAlias` be a good name for this?

Reply:
You're right. I think `expandAlias` would be a better name for `substituteInAlias`, but I can't think of a good name for the current `expandAlias`, since it's a bit weird with its recurse parameter. I'll think about it later, but you can disregard this comment. I can do the renaming myself, later.